thermal/thermal-daemon/thermald-lite.te

```diff
@@ -0,0 +1,10 @@
+# thermald-lite seclabel is specified in init.<board>.rc
+type thermald-lite, domain;
```

Comment:
please refer to this te file and add "domain_trans(init, rootfs, thermal_lite);". And you'd better wrap the rules with recovery_only.
http://inlubt0291.iind.intel.com:8080/source/xref/Android-S-GP2.3/device/intel/sepolicy/thermal/dptf/thermal_lite.te?r=71253d20#7